### ISSUE TITLE
Incompatibility with phoenix_pubsub 1.0

### ISSUE THREAD

**Issue by merqlove** [phoenixframework/phoenix_pubsub_redis#18]
Supervisor:
```elixir
supervisor(Phoenix.PubSub.Redis, [:freeling, [host: "127.0.0.1", pool_size: 1, namespace: "freeling"]]),
```

Commands:
```elixir
Phoenix.PubSub.subscribe :freeling, "user:123"
Phoenix.PubSub.broadcast :freeling, "user:123", {:user_update, %{id: 123, name: "Shane"}}
```

Result:
```
** (UndefinedFunctionError) function Phoenix.PubSub.Local.broadcast/5 is undefined or private. Did you mean one of:

      * broadcast/6

    (phoenix_pubsub) Phoenix.PubSub.Local.broadcast(:freeling, 1, :none, "user:123", {:user_update, %{id: 123, name: "Shane"}})
```

**Comment by merqlove** [comment]
Seems it is not tested at all with new pubsub.

**Comment by merqlove** [comment]
Did workaround, all tests passed, except:

```elixir
  1) test node_name/1 returns the node name (Phoenix.PubSubTest)
     deps/phoenix_pubsub/test/shared/pubsub_test.exs:66
     ** (MatchError) no match of right hand side value: []
     stacktrace:
       lib/phoenix/pubsub.ex:287: Phoenix.PubSub.call/3
       deps/phoenix_pubsub/test/shared/pubsub_test.exs:67: (test)
```
https://github.com/merqlove/phoenix_pubsub_redis

**Comment by chrismccord** [comment]
This will be fixed in #14

**Comment by merqlove** [comment]
Thanks!
